src/sage/combinat/words/morphism.py

```diff
@@ -1725,11 +1727,19 @@ def is_uniform(self, k=None):
             False
             sage: tau.is_uniform(k=2)
             True
+
+        TESTS:
+
+            sage: phi = WordMorphism('')
+            sage: phi.is_uniform()
+            False
```

Comment:
I know that this is the existing behavior, but is it correct? After all, we have
```
sage: phi.is_uniform(0)
True
```

Reply:
indeed, I changed the answer to True